packages/plugin-infra/src/web3/useWeb3.ts

```diff
@@ -10,7 +10,7 @@ export function useWeb3<T extends NetworkPluginID>(pluginID?: T, options?: Web3H
     type GetWeb3 = (options?: Web3Helper.Web3ConnectionOptions<T>) => Promise<Web3Helper.Web3<T>>
 
     const { Connection } = useWeb3State(pluginID)
-    const chainId = useChainId(pluginID)
+    const chainId = useChainId(pluginID, options?.chainId)
```

Comment:
You will finally get the `chainId` to send to `getWeb3` in line #23.

Reply:
> You will finally get the `options.chainId` to send to `getWeb3` in line #23. So here is not necessary, right?

Indeed.